libnmstate/netapplier.py

```diff
@@ -63,9 +63,20 @@ def apply(desired_state, verify_change=True, commit=True, rollback_timeout=60):
     validator.validate_dns(desired_state)
     validator.validate_vxlan(desired_state)
 
+    not_desired_state_vlans = _prepare_vlan_ifaces_for_mtu_change(
```

Comment:
I'd rename `not_desired_state_vlans`to something more in line w/ reality, like `vlans_with_mtu_changes` or something like that.

Reply:
Done! Thanks.